### PR TITLE
Script improvements

### PR DESCRIPTION
**Pull request by nathanchance**
Commit messages and changes are fairly self explanatory :)

### CODE REVIEW

File: scripts/check-binaries.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    fi
+}
+
+BINARIES=( "clang" "ld.lld" "llvm-ar" "llvm-nm" )
+for BINARY in "${BINARIES[@]}"; do
+    test_command "${BINARY}-${LLVM_VERSION}"
+done
+
+QEMU_PREFIXES=( "arm" "aarch64" "i386" "mips" "mipsel" "ppc" "ppc64" "x86_64" )
+for QEMU_PREFIX in "${QEMU_PREFIXES[@]}"; do
```
Comment (nickdesaulniers) [inline scripts/check-binaries.sh]: technically these are suffixes; the binutils ones below are prefixes.

Comment (nathanchance) [reply scripts/check-binaries.sh]: Done, thanks!

File: Makefile [hunk 2]
```diff
@@ -16,28 +16,10 @@ release:
...
-	$(DOCKER) run --rm -ti $(REPO):$(LATEST_TAG) qemu-system-x86_64 --version
-	@echo
-	$(DOCKER) run --rm -ti $(REPO):$(LATEST_TAG) qemu-system-ppc --version
-	@echo
-	$(DOCKER) run --rm -ti $(REPO):$(LATEST_TAG) qemu-system-ppc64 --version
-	@echo
-	$(DOCKER) run --rm -ti $(REPO):$(LATEST_TAG) qemu-system-mips --version
-	@echo
-	$(DOCKER) run --rm -ti $(REPO):$(LATEST_TAG) qemu-system-mipsel --version
+	$(DOCKER) run --rm -ti --env LLVM_VERSION=$(LLVM_VERSION) --mount type=bind,source=$(shell pwd),target=/dockerimage $(REPO):$(LATEST_TAG) bash /dockerimage/scripts/check-binaries.sh
```
Comment (nickdesaulniers) [inline Makefile]: I honestly have no idea how docker's command line interface works. LGTM? 🤷‍♂ 😕 ❓ ❔

**Comment by nathanchance** [comment]
Thanks for the review :)

**Pull request merged**